### ISSUE TITLE
Create a loading indicator for the NavBar elements

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-code-manager#1187]
As discussed https://github.com/mozilla/addons-code-manager/issues/114#issuecomment-547967305

STR:
1. Load a Browser page which contains reviewer comments https://code.addons-dev.allizom.org/en-US/browse/390151/versions/1688302/?messageUid=0f2fa4cbd8354a6fb1020f5f7ec2369e&path=mozilla-recommendation.json
2. Observe the NavBar while the page is loading.

Expected result:
A loading indicator is available for `Comments` and `Comment Summary`. 

Actual result:
`Comments` and `Comment Summary` appear with a small delay. 

Notes:
This issue is reproducible with the reviewer comments too. Same fix should be applied but perhaps after https://github.com/mozilla/addons-code-manager/issues/114 lands. 

the Nav Bar

![the elements](https://user-images.githubusercontent.com/33448286/67933257-1a06d180-fbce-11e9-9196-ee1b49d8f9d1.gif)

the Comments

![the comments](https://user-images.githubusercontent.com/33448286/67933101-d90ebd00-fbcd-11e9-8508-f77a0c870a09.gif)

**Comment by kumar303** [comment]
@ioanarusiczki ah, I also noticed that the API could be faster here. I filed https://github.com/mozilla/addons-server/issues/12779

Here are some thoughts on the loading indicator
- It's currently impossible to know if a version has saved comments or not before making an API request for those comments. Thus, it wouldn't make sense (or would it?) to show a loading indicator for the case where the version does not have comments. This needs some thought.
- Maybe there is a way to patch the initial get version API response to indicate whether saved comments exist or not for the current user. This would solve the problem above.

**Comment by ioanarusiczki** [comment]
@kumar303 Indeed, I see that the response timings increase with the number of comments made. 


12 comments from AMO dev 

![Frontend timing](https://user-images.githubusercontent.com/33448286/68110677-1b017100-fef6-11e9-8a08-ce0d270447de.png)

12 comments from Postman - the API response :

![Response for 12 comments](https://user-images.githubusercontent.com/33448286/68110724-30769b00-fef6-11e9-86cc-9302acf11c0f.png)
